interface/forms/observation/new.php

```diff
@@ -114,7 +114,7 @@ function sel_code(id)
                 id = id.split('tb_row_');
                 var checkId = '_' + id[1];
                 document.getElementById('clickId').value = checkId;
-                dlgopen('<?php echo $GLOBALS['webroot'] . "/interface/patient_file/encounter/" ?>find_code_popup.php?codetype=LOINC,PHIN Questions', '_blank', 700, 400);
+                dlgopen('<?php echo $GLOBALS['webroot'] . "/interface/patient_file/encounter/" ?>find_code_popup.php?"codetype=LOINC, PHIN Questions"', '_blank', 400, 400);
```

Comment:
i think you just need to urlencode to original for this to work:
```
dlgopen('<?php echo $GLOBALS['webroot'] . "/interface/patient_file/encounter/" ?>find_code_popup.php?codetype=' + encodeURIComponent('LOINC,PHIN Questions'), '_blank', 700, 400);
```

Reply:
hi @bradymiller Sorry it was a mistake, you may have a look